nucliadb_node/src/writer/grpc_driver.rs

```diff
@@ -101,23 +101,19 @@ impl NodeWriter for NodeWriterGRPCDriver {
         self.instrument(&request);
 
         info!("gRPC delete_shard {:?}", request);
-
+        // Deletion does not requiere for the shard
```

Comment:
Little typo here `requiere` => `require`

Reply:
Ouch true! Thanks for noticing, fixed it at [ad5a4c3](https://github.com/nuclia/nucliadb/pull/462/commits/ad5a4c34b67331b5357922d335e22f3c09505558)